code/modules/pda/PDA.dm

```diff
@@ -299,12 +299,15 @@ GLOBAL_LIST_EMPTY(PDAs)
 	else if(istype(C, /obj/item/pen))
 		var/obj/item/pen/O = locate() in src
 		if(O)
-			to_chat(user, "<span class='notice'>There is already a pen in \the [src].</span>")
-		else
-			user.drop_item()
-			C.forceMove(src)
-			to_chat(user, "<span class='notice'>You slide \the [C] into \the [src].</span>")
-			playsound(src, 'sound/machines/pda_button1.ogg', 50, TRUE)
+			to_chat(user, "<span class='notice'>There is already a pen in [src].</span>")
```

Comment:
does para not have \the ?

Reply:
we have \the, but it isn't needed because it is implied by [src]

Reply:
oh awesom :)

Reply:
good luck with the pr